### DISCUSSION TITLE
How do I query dataview?

### DISCUSSION THREAD

**Discussion by Programazing** [ivan-lednev/obsidian-day-planner#378] [Q&A] [answered]
I'm missing something about this feature.

I've read the README and issue #280 but still need help understanding.

I've set the filter to #tasks and given my pre-existing Tasks view the tasks tag, but I do not see the tasks scheduled for the next day.

It pulls from the daily note just fine.

![image](https://github.com/ivan-lednev/obsidian-day-planner/assets/79870333/1845b679-ce4a-40bc-b84d-96fc655e3d3d)

![tasks-view-with-calendar](https://github.com/ivan-lednev/obsidian-day-planner/assets/11393826/6deabcca-2ff2-41fd-8afa-5ce45d84e5a8)

I also have a lot of task views below the tasks at the top of the note. Such as:

\```tasks
not done
due today
\```

**Comment by ivan-lednev** [comment, marked as answer]
Hello. The issue here is that you don't use the `scheduled` property (⏳ 2024-06-01). Currently, this is the only one that's supported.
